include/cuda/std/detail/libcxx/include/algorithm

```diff
@@ -3302,6 +3336,7 @@ partition_copy(_InputIterator __first, _InputIterator __last,
 // partition_point
 
 template<class _ForwardIterator, class _Predicate>
+_LIBCUDACXX_INLINE_VISIBILITY
```

Comment:
We probably also want `_LIBCUDACXX_EXECUTION_SPACE_SPECIFIER` for the ones here and above? I assume this is where you invented the new macro.

Reply:
I'm wary to do that, when building source files in libcxx there are places where some extra ABI flags are set that may matter. The ones I've marked are mainly internal and inline.

Putting `_LIBCUDACXX_INLINE_VISIBLITY` on those causes them to fail to build due to the linkage flags.

e.g.
```
#if __has_attribute(internal_linkage)
#  define _LIBCUDACXX_INTERNAL_LINKAGE __attribute__ ((internal_linkage))
#else
#  define _LIBCUDACXX_INTERNAL_LINKAGE _LIBCUDACXX_ALWAYS_INLINE
#endif
```